lib/src/utils/event_localizations.dart

```diff
@@ -240,6 +240,11 @@ abstract class EventLocalizations {
         event.senderFromMemoryOrFallback.calcDisplayname(i18n: i18n)),
     EventTypes.CallCandidates: (event, i18n, body) => i18n.sentCallInformations(
         event.senderFromMemoryOrFallback.calcDisplayname(i18n: i18n)),
+    EventTypes.GroupCallMember: (event, i18n, body) {
+      final memberships = event.content.tryGetList('memberships');
```

Comment:
also needs an expired check. I would just use `getCallMembershipsFromEvent` and do a .isExpired

Reply:
done